lua/terrortown/lang/en.lua

```diff
@@ -1952,3 +1952,9 @@ L.status_off = "off"
 
 L.crowbar_help_primary = "Attack"
 L.crowbar_help_secondary = "Push players"
+
+-- 2023-10-27
+L.help_HUD_enable_description = [[
+Some HUD elements like the key helper or sidebar show detailed information when the scoreboard is open. This can be disabled to reduce clutter.]]
+L.label_HUD_enable_description = "Enable descriptions when scoreboard is open"
+L.label_HUD_enable_box_blur = "Enable UI box background blur"
```

Comment:
add those to the other languages too

Reply:
I though I could leave them away because the tool can do this prior to release, hm

Reply:
I think at some point we might add your tool as another CI check and we should add this in as soon as it hits the main branch, so that new contributions to translations can directly benefit from this before a new update is released.

Reply:
ok, I will add it then manually for now